### DISCUSSION TITLE
How to exclude database name from tables in diagram

### DISCUSSION THREAD

**Discussion by flcdrg** [schemacrawler/SchemaCrawler#769] [Q&A] [answered]
Running SchemaCrawler against a SQL Server database to generate a diagram, and all the tables are prefixed with the database name. eg. database.schema.tablename.

![image](https://user-images.githubusercontent.com/384747/174974770-4f621dd3-2053-4472-800b-f05d0b8e7455.png)

Is there a way to just have schema.tablename?

**Comment by sualeh** [comment, marked as answer]
@flcdrg David, please try the `--portable-names` option from the command-line. This will only show the table name. Currently, there is no way to have just schemaname.tablename. Please let me know if this is a required feature for your use case.

**Reply by flcdrg** [reply]
yeah, `--portable-names` works as you describe. 

It would be nice to have the option of dropping the database prefix - it feels redundant given that all the objects (in my case at least) are in the same database.

**Reply by sualeh** [reply]
@flcdrg David, that is what I figured. In most cases, the functionality offered by `--portable-names` is sufficient.
